src/backend/joanie/core/utils/sentry.py

```diff
@@ -0,0 +1,99 @@
+"""Sentry utilities."""
+
+import json
+
+from django.conf import settings
+
+from cryptography.fernet import Fernet, InvalidToken
+
+from joanie.core.fields.schedule import OrderPaymentScheduleEncoder
```

Comment:
What do you think to rename this encoder is a more generic manner ?

Reply:
I'm not sure abut this, as OrderPaymentScheduleEncoder is clearly dedicated to the payment_schedule attribute.
If you find strange to have this import here, I'd rather use `class SentryEncoder(DjangoJSONEncoder):` and add Money management here.